blog/2022-03-01-temporal-rest.md

```diff
@@ -0,0 +1,239 @@
+---
+tags:
+  - temporal
+  - community
+posted_on_: 2022-03-01T00:00:00Z
+slug: introducing-temporal-rest
+title: 'Introducing Temporal-Rest'
+author: Valeri Karpov
+author_title: Community Member
+author_image_url: https://avatars.githubusercontent.com/u/1620265?v=4
+release_version: V1.15
+---
+
+<!--truncate-->
+
+You can use Temporal's long-lived Workflows to build [REST APIs without a conventional database](https://docs.temporal.io/blog/build-an-ecommerce-app-with-temporal-part-4-rest-api).
+While you can build a REST API on top of Temporal Workflows yourself, we built [`temporal-rest`](https://www.npmjs.com/package/temporal-rest) to make creating a RESTful API for your Workflows a one-liner with [ExpressJS](https://expressjs.com/).
```

Comment:
Maybe change "we" to "I" here to avoid implying that this is an official Temporal project?

Reply:
```suggestion
While you can build a REST API on top of Temporal Workflows yourself, I built [`temporal-rest`](https://www.npmjs.com/package/temporal-rest) to make creating a RESTful API for your Workflows a one-liner with [ExpressJS](https://expressjs.com/).
```